modules/monitoring-granting-users-permissions-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="granting-users-permissions-for-core-platform-monitoring_{context}"]
+= Granting users permissions for core platform monitoring
+
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
+
+You can also grant developers and other users different permissions for core platform monitoring:
+
+* To monitor core platform monitoring components.
+* To query the cluster resource metrics API.
+* To monitor or manage alerts and silences.
+* To access Alertmanager component.
+
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
```

Comment:
should we state explicitly if a row is a role or cluster role?

Reply:
I hope it will be clear with the addition of the third column here :+1: